src/shared/components/common/moment-time.tsx

```diff
@@ -36,15 +36,32 @@ export class MomentTime extends Component<MomentTimeProps, any> {
     return line;
   }
 
+  isRecentlyUpdated(): boolean {
+    if (this.props.updated) {
+      const published = new Date(this.props.published);
+      const updated = new Date(this.props.updated);
+      const updateLimit = addMinutes(published, 5);
+      return isBefore(updated, updateLimit);
+    }
+    return false;
+  }
+
+  get updatedTime(): string {
+    if (this.isRecentlyUpdated()) {
+      return "";
+    }
+    return this.props.updated || "";
+  }
```

Comment:
use an else block, and don't return empty strings. Use undefined or null for missing values.